src/main/java/hw_3/Task3.java

```diff
@@ -0,0 +1,22 @@
+package hw_3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        // ДЗ Масиви та рядки
+        /*Створіть рядок "Hello world!"
```

Comment:
There should be no Latin words

Reply:
I wrote it for myself! I could have just deleted it, but okay - I'll translate it. Fixed

Reply:
not